test/conformance/context/urContextCreate.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#include <uur/fixtures.h>
+
+using urContextCreateTest = uur::urDeviceTest;
+
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urContextCreateTest);
+
+TEST_P(urContextCreateTest, Success) {
+  ur_context_handle_t context = nullptr;
+  ASSERT_SUCCESS(urContextCreate(1, &device, &context));
```

Comment:
should we have test with more than 1 device?

Reply:
Multi-device contexts are still a work in progress at the moment - due to limitations with the CUDA backend only supporting one device per context. I'll create an issue to add conformance tests for this once its ready.

Reply:
I've added a test for this - but this will likely change in near future.